### ISSUE TITLE
Detects geometry differences where there is none?

### ISSUE THREAD

**Issue by atorger** [matthiasfeist/nvdb-osm-diff-map#2]
Dorotea, Storuman, Vilhelmina, Lycksele municipalities has all geometry updated with NVDB. It's not an exact copy and there are some minor differences here and there, but it seems like the "missing road or large geometry differences" feature is overstating the differences, and sometimes marks a red hexagon (usually isolated on its own) in places where there is no difference at all, which you can see when you zoom in max.

**Comment by atorger** [comment]
More info, Dorotea, Vilhelmina and Storuman was imported when I used stronger simplification of the geometry than Lycksele Övertorneå and Sorsele which I imported after we changed the simplification algorithm to max 0.2 meter deviation on all geometry. That may affect the difference detection I suppose.

I'm not sure what a suitable threshold should be for difference detection, but I think one could have it quite high, like 5 meters or so to minimize false positives and focus on the really bad cases which there are quite a few still as many roads are mapped after old landsat images.
